helm_deploy/prisoner-offender-search/restore.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -e
+
+# Snapshot restores only happen every other week, crontab doesn't really support this
+week=$(date +%U)
+if [[ $(("$week" % 2)) == 0 ]]; then
+  echo "Not running restore this week, restores only happens on odd week numbers, check back next week."
+  exit 0
+fi
+
+# Check that we can connect to elastic search
+if ! OUTPUT=$(http --check-status --ignore-stdin "${ELASTICSEARCH_PROXY_URL}"); then
+  echo -e "\nUnable to talk to elastic search API"
+  echo "$OUTPUT"
+  exit 1
+fi
+
+http_es() { http --stream --check-status --ignore-stdin --timeout=600 "$@"; }
+
+# we would normally restore to a different namespace than the one we've taken the snapshot from
+SNAPSHOT_NAMESPACE="${NAMESPACE_OVERRIDE:-${NAMESPACE}}"
+ENDPOINT_SNAPSHOT_NAMESPACE="$ELASTICSEARCH_PROXY_URL/_snapshot/$SNAPSHOT_NAMESPACE"
+ENDPOINT_LATEST="$ENDPOINT_SNAPSHOT_NAMESPACE/latest"
+INDICES="prisoner-search-a,prisoner-search-b,offender-index-status"
```

Comment:
had to specify which indices we operate on as there are some kibana ones that get created by the application automatically and want to ignore those as otherwise can't restore